.golangci.yml

```diff
@@ -118,6 +118,7 @@ linters-settings:
         allow:
           - github.com/grafana/grafana/pkg/apimachinery
           - github.com/grafana/grafana/pkg/apiserver
+          - github.com/grafana/grafana/pkg/storage/unified/resource
```

Comment:
@toddtreece -- I added this because this exposes the interfaces used to talk to the resource server